[12.0][FIX] purchase_open_qty

AdriaGForgeFlow opened the pull request:
This module introduces two filters, in the Purchase Order Line search view, which use the fields:
* pending_qty_to_invoice
* pending_qty_to_receive

However, those fields are not defined in the model so every time we try to use them it gives an error. This PR fixes that issue by adding those two fields to the purchase.order.line model.

cc ~ @ForgeFlow

AaronHForgeFlow reviewed (approved):
:+1: Code Review. LGTM.

eantones committed:
[FIX] purchase_open_qty: not able to filter by pending/receiving quantity at line level

Wrong filter fields at line level, the module is trying to use the
computed fields of the order instead of the line ones.

AaronHForgeFlow commented:
Maybe the new fields are not necessary, but changing the field names :thinking: https://github.com/OCA/purchase-workflow/pull/910

eantones commented:
@AdriaGForgeFlow  you are duplicating code and introducing an unnecessary computation, you don't really need a computed field at line level because you already have the data stored on qty_to_invoice and qty_to_receive. In my opinion the approach on https://github.com/OCA/purchase-workflow/pull/910 is more efficient, simpler and has much less code so much less risk to introduce bugs.
cc @AaronHForgeFlow 
Thanks! :-)

AdriaGForgeFlow commented:
> @AdriaGForgeFlow you are duplicating code and introducing an unnecessary computation, you don't really need a computed field at line level because you already have the data stored on qty_to_invoice and qty_to_receive. In my opinion the approach on #910 is more efficient, simpler and has much less code so much less risk to introduce bugs.
> cc @AaronHForgeFlow
> Thanks! :-)

Will do, thanks for the suggestion

AdriaGForgeFlow commented:
@AaronHForgeFlow @eantones  Can you review again?

AaronHForgeFlow reviewed (approved):
Still LGTM

HviorForgeFlow commented:
/ocabot merge path

HviorForgeFlow commented:
/ocabot merge patch
